doc/Tutorials.xml

```diff
@@ -181,12 +181,108 @@ There are various examples of packages using &AutoDoc; for only
 this purpose, e.g. <Package>IO</Package> and <Package>orb</Package>.
 <P/>
 
-<!--
-TODO: Explain how to mix AutoDoc content with existing &GAPDoc; XML files.
--->
+<Subsection Label="Tut:IntegrateExisting:EverythingThere">
+  <Heading>Using &AutoDoc; on a complete &GAPDoc; manual</Heading>
+
+Suppose you already have a complete XML manual, with some main and title
+XML files and some documentation for operations distributed over
+all your <F>.g{,d,i}</F> files. Even then &AutoDoc; can do something for you.
+Suppose your main xml file is named <F>PACKAGENAME.xml</F> and is in the
+<F>/doc</F> subfolder of your package. Then calling
+<Listing>
+LoadPackage( "AutoDoc" );
+AutoDoc( : autodoc := false );
+</Listing>
+will rebuild your manual. The main point in favor of doing is that you do
+not have to keep a list of all the <F>.g{,d,i}</F> files you have manual entries in.
+<P/>
+But there is more. &AutoDoc; can create a <F>maketest.g</F> file, which uses the
+examples in your manual to test your package. This can be achieved via
+<Listing>
+LoadPackage( "AutoDoc" );
+AutoDoc( : autodoc := false, maketest := true );
+</Listing>
+Now the file <F>maketest.g</F> appears in your package directory, and
+<Listing>
+gap maketest.g
+</Listing>
+test the examples from your manual.
+</Subsection>
 
-</Section>
+<Subsection Label="Tut:IntegrateExisting:GapDocOptions">
+<Heading>Setting different &GAPDoc; options</Heading>
 
+Sometimes, the default values for the &GAPDoc; command used by &AutoDoc;
+may not be suitable for your manual.
+<P/>
+Suppose you have your main xml file called <F>main.xml</F>. Then you can tell &AutoDoc;
+to use this file as the main xml file via
+<Listing>
+LoadPackage( "AutoDoc" );
+AutoDoc( : autodoc := false,
+           gapdoc := rec( main := "doc/main.xml" ) );
+</Listing>
+<P/>
+Also, it might happen that your <F>g.{,d,i}</F> files are not in somehow standard named
```

Comment:
`are not in somehow standard named subdirectories` -- first off, this sounds wrong (grammar?), secondly, it's meaning seems unclear. Perhaps start by stating explicitly where GAP searches, and then covering the exception?

> As explained above, by default AutoDoc scans all .g, .gd and .gi files it can find inside of your package root directory, as well as the subdirectories gap, lib, examples and examples/doc. If you keep source files with documentation in other directories, you can adjust the list of directories AutoDoc scans via the scan_dirs option. The following example illustrates this by instructing AutoDoc to only search in the subdirectory package_sources of the package root directory.